Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
+
+RUN apk --update add curl
+
+RUN curl -fsSL https://github.com/mozilla-conduit/phabricator-extensions/archive/${SHA}.tar.gz -o extensions.tar.gz \
+      && tar xzf extensions.tar.gz \
+      && rm extensions.tar.gz
+
+VOLUME ["extensions"]
```

Comment:
Needs to be: 

VOLUME ["/app/phabricator/src/extensions"]

So you also need to untar extensions.tar.gz to that same location.